### ISSUE TITLE
Suggestion for model.update is missing the save

### ISSUE THREAD

**Issue by oliverklee** [bbatsov/rubocop#2808]
In one of our projects (Ruby 2.3.0 with TargetRubyVersion: 2.3, Rails 4.2.5.1), we have the following line in one of our controller specs:

```ruby
    context 'with admin logged with attendee dashboard saved' do
      before do
        admin.update(dashboard: :attendee)
        sign_in admin
      end

      it 'redirects to the attendee dashboard' do
        get :index

        expect(response).to redirect_to attendee_dashboard_path
      end
    end
```

RuboCop complains about the update call:

```
spec/controllers/home_controller_spec.rb:89:9: C: Use admin[:dashboard] = :attendee instead of admin.update(dashboard: :attendee).
        admin.update(dashboard: :attendee)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

However, the suggestion is not correct. Only assigning the attribute (which is not a relation) will not save the model. So the admin.save needs to be added to the suggestion.

**Comment by bbatsov** [comment]
This is fixed in 0.37.1.

**Comment by oliverklee** [comment]
Wow, that was fast. Thanks! :heart:
